resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "Tells r_c to toughen up"
+        ],
+        "random_status_constraint": [
+            "apprentice"
+        ],
+        "main_trait_constraint": [
+            "cold"
+        ],
+        "random_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "compassionate_warrior",
+        "thoughts": [
+            "Spends time with a grieving Clanmate today",
+            "Is patiently listening to someone talk about things they are unquestionably wrong about and dying inside",
+            "Let {PRONOUN/m_c/poss} Clanmate have the last piece of fresh-kill on the pile this morning",
+            "Is noticing with joy how well the Clan is looking after one another as of late",
+            "Is listening to a Clanmate's struggles with love",
+            "Lets a Clanmate cry on {PRONOUN/m_c/poss} shoulder",
+            "Makes sure to check on a Clanmate who's been having a rough time",
+            "Quickly forgives somebody after they snapped at {PRONOUN/m_c/object}",
+            "Spends some extra time thanking StarClan for the prey today",
+            "Is waiting until everyone has eaten to take from the fresh-kill pile",
+            "Is patiently standing guard outside of camp",
+            "Is winning a staring contest against a Clanmate",
+            "Is listening to a story {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} heard many, many times",
+            "Is listening to older cats gripe about their day",
+            "Is looking up, watching the clouds roll by",
+            "Is watching the breeze blow around the camp",
+            "Is listening to the grass sway",
+            "Is listening closely to the sounds of the forest",
+            "Is waiting to watch the sun rise",
+            "Is waiting to watch the sun set",
+            "Offers to help clanmates with their duties",
+            "Is grooming, grooming, grooming away...",
+            "Smiles at some kit's cheap prank, knowing they'll grow out of it soon",
+            "Is the only cat not bothered by a snoring Clanmate",
+            "Is delighted by the slow passage of time",
+            "Feels entirely unlike {PRONOUN/m_c/self} when {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} bothered by something small",
+            "Is laughing at a Clanmate's joke, even though {PRONOUN/m_c/subject} didn't find funny",
+            "Helped to gather herbs all day",
+            "Gives some plump fresh-kill to a Clanmate instead",
+            "Offers a helping paw, even after a night of bad sleep",
+            "Feels {PRONOUN/m_c/poss} kindness might be taken advantage of",
+            "Is listening to the woes of a fellow Clanmate",
+            "Notices an apprentice struggling with a task and offers {PRONOUN/m_c/poss} help",
+            "Is doing some apprentice tasks around camp, to help lighten the load",
+            "Is sharing tongues with friends",
+            "Is keeping the injured cats company",
```

Comment:
i don't believe we have a way to specify that clan has to have injured cats in it, so probably just remove this one so that nothing incongruent happens